.evergreen/evergreen.yml.in

```diff
@@ -2,8 +2,8 @@
 <%
 const path = require('path');
 
-const NODE_JS_VERSION_14 = '14.19.2';
-const NODE_JS_VERSION_16 = '16.15.0';
+const NODE_JS_VERSION_14 = '14.20.0';
+const NODE_JS_VERSION_16 = '16.16.0';
```

Comment:
These are security releases, and while there should be little to no impact on mongosh [according to the vulnerability descriptions](https://nodejs.org/en/blog/vulnerability/july-2022-security-releases/), it’s probably a good idea to bump this version anyway. Doing it here is probably easier than a full new PR, can split it out if necessary. :)